### PR TITLE
add TrafficStrategyConfigurationYamlSwapper test

### LINKED ISSUE #16292: Add more unit test for TrafficStrategyConfigurationYamlSwapper
Hi community,

This issue is for [OpenForce](https://github.com/apache/shardingsphere/discussions/12683#discussioncomment-2139441)

We need to add more unit test for TrafficStrategyConfigurationYamlSwapper.

- [ ] TrafficStrategyConfigurationYamlSwapper#swapToYamlConfiguration
- [ ] TrafficStrategyConfigurationYamlSwapper#swapToObject

### PR DESCRIPTION
**Pull request by weihubeats**
Fixes #16292.

Changes proposed in this pull request:
- add TrafficStrategyConfigurationYamlSwapperTest

### CODE REVIEW

File: shardingsphere-kernel/shardingsphere-traffic/shardingsphere-traffic-core/src/test/java/org/apache/shardingsphere/traffic/yaml/swapper/TrafficStrategyConfigurationYamlSwapperTest.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    private YamlTrafficStrategyConfiguration createYamlTrafficStrategyConfiguration() {
+        YamlTrafficStrategyConfiguration yamlTrafficStrategyConfiguration = new YamlTrafficStrategyConfiguration();
+        yamlTrafficStrategyConfiguration.setName(NAME);
+        yamlTrafficStrategyConfiguration.setLabels(LABELS);
+        yamlTrafficStrategyConfiguration.setAlgorithmName(ALGORITHM_NAME);
+        yamlTrafficStrategyConfiguration.setLoadBalancerName(LOAD_BALANCER_NAME);
+        return yamlTrafficStrategyConfiguration;
+    }
+
```
Comment (strongduanmu) [inline shardingsphere-kernel/shardingsphere-traffic/shardingsphere-traffic-core/src/test/java/org/apache/shardingsphere/traffic/yaml/swapper/TrafficStrategyConfigurationYamlSwapperTest.java:79]: @weihubeats Please remove this useless blank line.

File: shardingsphere-kernel/shardingsphere-traffic/shardingsphere-traffic-core/src/test/java/org/apache/shardingsphere/traffic/yaml/swapper/TrafficStrategyConfigurationYamlSwapperTest.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        assertThat(trafficStrategyConfiguration.getLoadBalancerName(), is(LOAD_BALANCER_NAME));
+    }
+
+    private YamlTrafficStrategyConfiguration createYamlTrafficStrategyConfiguration() {
+        YamlTrafficStrategyConfiguration yamlTrafficStrategyConfiguration = new YamlTrafficStrategyConfiguration();
+        yamlTrafficStrategyConfiguration.setName(NAME);
+        yamlTrafficStrategyConfiguration.setLabels(LABELS);
+        yamlTrafficStrategyConfiguration.setAlgorithmName(ALGORITHM_NAME);
+        yamlTrafficStrategyConfiguration.setLoadBalancerName(LOAD_BALANCER_NAME);
+        return yamlTrafficStrategyConfiguration;
```
Comment (strongduanmu) [inline shardingsphere-kernel/shardingsphere-traffic/shardingsphere-traffic-core/src/test/java/org/apache/shardingsphere/traffic/yaml/swapper/TrafficStrategyConfigurationYamlSwapperTest.java:77]: Rename yamlTrafficStrategyConfiguration to result may be better.

**Review by strongduanmu — CHANGES_REQUESTED** [review]

**Comment by weihubeats** [comment]
all processed

**Comment by strongduanmu** [comment]
> all processed

@weihubeats Good job, I will merge this pr after ci success.

**Comment by weihubeats** [comment]
Good looking forward to the opportunity to cooperate again

**Review by strongduanmu — APPROVED** [review]
@weihubeats Thank you, merged.

**Pull request merged**